src/test/kotlin/org/jetbrains/research/transformations/util/PythonTransformationsTest.kt

```diff
@@ -0,0 +1,48 @@
+import com.github.gumtreediff.gen.python.PythonTreeGenerator
+import com.github.gumtreediff.tree.TreeContext
+import org.jetbrains.research.transformations.util.ParserSetup
+import org.jetbrains.research.transformations.util.Util
+import org.jetbrains.research.transformations.util.toXMLWithoutRoot
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.TestInstance
+import java.io.File
+import java.util.logging.Logger
+import kotlin.reflect.KFunction
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+open class PythonTransformationsTest {
+    protected val LOG = Logger.getLogger(javaClass.name)
+
+    protected val XMLTreeFileName = "${Util.getTmpPath()}/test.xml"
+
+    companion object {
+        fun getResourcePath(resourceName: String, cls: KFunction<PythonTransformationsTest>): String {
+            return cls.javaClass.getResource(resourceName).path
+        }
+    }
+
+    @BeforeAll
+    internal fun beforeAll() {
+        LOG.info("beforeAll called")
+        ParserSetup.checkSetup()
+    }
+
+    protected fun getSourcePythonCode(ctx: TreeContext, XMLDstPath: String): String {
+        val treeSerializer = ctx.toXMLWithoutRoot()
+        treeSerializer.writeTo(XMLDstPath)
+        return Util.runProcessBuilder(ParserSetup.getCommandForInverseParser(XMLDstPath))
+    }
+
+    protected fun transformCode(inFile: File, outFile: File, transformation: (TreeContext, Boolean) -> Unit){
```

Comment:
Let's rename it to represent the fact that it asserts something. What do you think about "assertCodeTransformation"?

Reply:
Great idea, thanks!